design/templates/solution_design.md

```diff
@@ -13,37 +19,95 @@
 |          |                 |
 |          |                 |
 
-## Useful links
+## Useful Links
 [//]: # "Add links that may be useful for the reader"
 
+|  **Name**   | **Link** |
+|-------------|----------|
+| Feature Doc |          |
+| Issue       |          |
+
 ## Background
 [//]: # "Give relevant background for the designed feature. What is the motivation for this solution?"
 
-## Issue description
+## Issue Description
 [//]: # "Elaborate on the issue you are writing a solution for"
```

Comment:
@rafis3 what is the difference between this and the "background" section? they seem very similar to me

Reply:
Background is meant to put the reader in context. For example, for a modified Kubernetes authenticator, in the background I would write what we currently have and what is it used for, just in a few short sentences. Then in the issue I'd explain what are the gaps, issues with the current state that we want to solve. Then in the solution, I'd explain how it would be solved and why the suggestion solves the issue.